src/containers/Header/Header.tsx

```diff
@@ -2,15 +2,19 @@ import classNames from 'classnames';
 import { LayoutContainer } from 'components/LayoutContainer/LayoutContainer';
 import { Menu } from 'components/Menu/Menu';
 import { UserMenu } from 'components/UserMenu/UserMenu';
+import { getAvatarUrl } from 'helpers/getAvatarUrl';
 import Logo from 'images/logo.svg';
 import { useMemo } from 'react';
 import { NavLink } from 'react-router-dom';
+import { profileApi } from 'services/redux';
+import { UserResponseKeys } from 'types/Api';
 import { AppRoutes } from 'types/AppRoutes';
 
 import styles from './Header.module.scss';
 import { HeaderProps } from './Header.types';
 
 export function Header({ className }: HeaderProps) {
+  const { data } = profileApi.useGetProfileQuery();
   const menuItems = useMemo(
```

Comment:
🧐
Если решишь, что в этом есть смысл, то так надо будет пройтись по всему проекту
```suggestion
  const { data } = profileApi.useGetProfileQuery();
  
  const menuItems = useMemo(
```